README.md

```diff
@@ -351,152 +351,454 @@ or equal to those of the children and the lowest key is in the root node
-│   └── wallsAndGates.java
-├── DepthFirstSearch
-│   ├── balancedBinaryTree.java
-│   ├── battleshipsInABoard.java
-│   ├── convertSortedArrayToBinarySearchTree.java
-│   ├── maximumDepthOfABinaryTree.java
-│   ├── numberOfIslands.java
-│   ├── populatingNextRightPointersInEachNode.java
-│   └── sameTree.java
-├── Design
-│   └── zigzagIterator.java
-├── DivideAndConquer
-│   ├── expressionAddOperators.java
-│   └── kthLargestElementInAnArray.java
-├── DynamicProgramming
-│   ├── bombEnemy.java
-│   ├── climbingStairs.java
-│   ├── combinationSumIV.java
-│   ├── countingBits.java
-│   ├── editDistance.java
-│   ├── houseRobber.java
-│   ├── paintFence.java
-│   ├── paintHouseII.java
-│   ├── regularExpressionMatching.java
-│   ├── sentenceScreenFitting.java
-│   ├── uniqueBinarySearchTrees.java
-│   └── wordBreak.java
-├── HashTable
-│   ├── binaryTreeVerticalOrderTraversal.java
-│   ├── findTheDifference.java
-│   ├── groupAnagrams.java
-│   ├── groupShiftedStrings.java
-│   ├── islandPerimeter.java
-│   ├── loggerRateLimiter.java
-│   ├── maximumSizeSubarraySumEqualsK.java
-│   ├── minimumWindowSubstring.java
-│   ├── sparseMatrixMultiplication.java
-│   ├── strobogrammaticNumber.java
-│   ├── twoSum.java
-│   └── uniqueWordAbbreviation.java
-├── LinkedList
-│   ├── addTwoNumbers.java
-│   ├── deleteNodeInALinkedList.java
-│   ├── mergeKSortedLists.java
-│   ├── palindromeLinkedList.java
-│   ├── plusOneLinkedList.java
-│   ├── README.md
-│   └── reverseLinkedList.java
-├── Queue
-│   └── movingAverageFromDataStream.java
-├── README.md
-├── Sort
-│   ├── meetingRoomsII.java
-│   └── meetingRooms.java
-├── Stack
-│   ├── binarySearchTreeIterator.java
-│   ├── decodeString.java
-│   ├── flattenNestedListIterator.java
-│   └── trappingRainWater.java
-├── String
-│   ├── addBinary.java
-│   ├── countAndSay.java
-│   ├── decodeWays.java
-│   ├── editDistance.java
-│   ├── integerToEnglishWords.java
-│   ├── longestPalindrome.java
-│   ├── longestSubstringWithAtMostKDistinctCharacters.java
-│   ├── minimumWindowSubstring.java
-│   ├── multiplyString.java
-│   ├── oneEditDistance.java
-│   ├── palindromePermutation.java
-│   ├── README.md
-│   ├── reverseVowelsOfAString.java
-│   ├── romanToInteger.java
-│   ├── validPalindrome.java
-│   └── validParentheses.java
-├── Tree
-│   ├── binaryTreeMaximumPathSum.java
-│   ├── binaryTreePaths.java
-│   ├── inorderSuccessorInBST.java
-│   ├── invertBinaryTree.java
-│   ├── lowestCommonAncestorOfABinaryTree.java
-│   ├── sumOfLeftLeaves.java
-│   └── validateBinarySearchTree.java
-├── Trie
-│   ├── addAndSearchWordDataStructureDesign.java
-│   ├── implementTrie.java
-│   └── wordSquares.java
-└── TwoPointers
-    ├── 3Sum.java
-    ├── 3SumSmaller.java
-    ├── mergeSortedArray.java
-    ├── minimumSizeSubarraySum.java
-    ├── moveZeros.java
-    ├── removeDuplicatesFromSortedArray.java
-    ├── reverseString.java
-    └── sortColors.java
-
-18 directories, 124 files
-```
+# Directory Tree
+
+## Airbnb
```

Comment:
^ Yea this is actually a good point. 

This might be overkill but I wonder if we can write a scrip that automates this on any new commits. I know github has an API and should be pretty easy to set something up, maybe w/ web hooks? I'm pretty sure it's also possible to have some type of "github cron" which runs on commits.